.github/workflows/update-website.yml

```diff
@@ -0,0 +1,50 @@
+name: Update website
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  rebaseline-tests:
+    name: Update website
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.EMSCRIPTEN_BOT_TOKEN }}
```

Comment:
Do you think we need to add this token to the emscripten-site repo in order to open PRs there?

@dschuff @kripken

Reply:
If the emscripten-site repo will accept PRs from any random person on github, then I don't see why emscripten-bot should be different. If we need to give emscripten-bot some permission to let CI on the site run, we could do that. But it shouldn't require actually putting the bot's token in the repo as a secret. That would only be needed if we want something to run as the bot.